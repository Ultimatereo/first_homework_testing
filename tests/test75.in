5
8044211 -8742607 5844750 -5752670 -8607444 426280
4365965 -243732 -8180174 -7232235 -8655007 -7644603
2234080 -4396930 -5162455 -6242260 -6187997 -4209138
9497259 4009299 6229915 7391503 5364018 -342724
-8468085 -9146389 3573912 -6998268 -2250451 1081460
