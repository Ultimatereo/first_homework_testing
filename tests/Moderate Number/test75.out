-0.722555
-0.648475
0.156954
2.08189
-1.35097
